[2025-02-22T13:50:37.785-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-22T13:50:37.807-0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fii_info.store.store_parquet manual__2025-02-22T16:50:16.319007+00:00 [queued]>
[2025-02-22T13:50:37.820-0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fii_info.store.store_parquet manual__2025-02-22T16:50:16.319007+00:00 [queued]>
[2025-02-22T13:50:37.821-0300] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-22T13:50:37.848-0300] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): store.store_parquet> on 2025-02-22 16:50:16.319007+00:00
[2025-02-22T13:50:37.853-0300] {standard_task_runner.py:72} INFO - Started process 28524 to run task
[2025-02-22T13:50:37.860-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fii_info', 'store.store_parquet', 'manual__2025-02-22T16:50:16.319007+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/info_Fii.py', '--cfg-path', '/tmp/tmpofi79xnh']
[2025-02-22T13:50:37.864-0300] {standard_task_runner.py:105} INFO - Job 45: Subtask store.store_parquet
[2025-02-22T13:50:37.923-0300] {task_command.py:467} INFO - Running <TaskInstance: fii_info.store.store_parquet manual__2025-02-22T16:50:16.319007+00:00 [running]> on host LAPTOP-3G01DURD.
[2025-02-22T13:50:38.016-0300] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fii_info' AIRFLOW_CTX_TASK_ID='store.store_parquet' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T16:50:16.319007+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-22T16:50:16.319007+00:00'
[2025-02-22T13:50:38.019-0300] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-22T13:50:38.032-0300] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/dags/info_Fii.py", line 55, in store_parquet
    cot = pd.read_csv(diretorio+doc, sep=';', encoding='latin-1',dtype=str,decimal=str)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1026, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 620, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1620, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1898, in _make_engine
    return mapping[engine](f, **self.options)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 93, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "parsers.pyx", line 449, in pandas._libs.parsers.TextReader.__cinit__
TypeError: object of type 'type' has no len()
[2025-02-22T13:50:38.041-0300] {taskinstance.py:1226} INFO - Marking task as FAILED. dag_id=fii_info, task_id=store.store_parquet, run_id=manual__2025-02-22T16:50:16.319007+00:00, execution_date=20250222T165016, start_date=20250222T165037, end_date=20250222T165038
[2025-02-22T13:50:38.069-0300] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-22T13:50:38.069-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 45 for task store.store_parquet (object of type 'type' has no len(); 28524)
Traceback (most recent call last):
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/dags/info_Fii.py", line 55, in store_parquet
    cot = pd.read_csv(diretorio+doc, sep=';', encoding='latin-1',dtype=str,decimal=str)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1026, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 620, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1620, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1898, in _make_engine
    return mapping[engine](f, **self.options)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 93, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "parsers.pyx", line 449, in pandas._libs.parsers.TextReader.__cinit__
TypeError: object of type 'type' has no len()
[2025-02-22T13:50:38.113-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-02-22T13:50:38.130-0300] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-22T13:50:38.131-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
